### PR TITLE
Add feature update notifier

### PR DESCRIPTION
**Pull request by andrew2003**
# Check List

- [x] Don't forget to squash commits into meaningful chunks before merging
- [x] Check every test passed.
- [x] Did you split imports into std, third parties, and custom parts?
- [x] Format your code.
  - [x] cargo fmt
  - [x] cargo clippy
  - [x] `taplo fmt --config taplo/taplo.toml`
- [x] Check unwrap, expect().
- [x] Check clone().
- [x] Check your commit messages.
- [x] Have you added meaningful comments.

### CODE REVIEW

File: src/bin/ornn.rs [hunk 1]
```diff
@@ -1,10 +1,12 @@
 use std::cmp::PartialEq;
+use std::time::Duration;
 
 use clap::CommandFactory;
 use clap::{Parser, Subcommand};
-use orn::const_values::get_constant_values;
-use orn::file_manager::FileManager;
-use orn::gen_const::gen_consts;
+use ornn::const_values::get_constant_values;
```
Comment (zk-steve) [inline src/bin/ornn.rs]:
```suggestion

use ornn::const_values::get_constant_values;
```

File: src/const_values.toml
Comment (zk-steve) [inline src/const_values.toml:1]: ?

File: src/gen_const.rs [hunk 2]
```diff
@@ -16,6 +19,16 @@ fn create_const_block(consts: &HashSet<String>, table: &HashMap<String, Constant
...
         return "".to_string();
     }
+
+    let table_keys: Vec<_> = table.keys().cloned().collect();
+    let consts_unused: Vec<_> = table_keys
+        .into_iter()
+        .filter(|item| !consts.contains(item))
+        .collect();
+    for e in consts_unused {
+        warn!("Unused: {}", e);
```
Comment (zk-steve) [inline src/gen_const.rs]: normal print

File: src/gen_const.rs [hunk 3]
```diff
@@ -16,6 +19,16 @@ fn create_const_block(consts: &HashSet<String>, table: &HashMap<String, Constant
     if consts.is_empty() {
         return "".to_string();
     }
+
+    let table_keys: Vec<_> = table.keys().cloned().collect();
+    let consts_unused: Vec<_> = table_keys
+        .into_iter()
+        .filter(|item| !consts.contains(item))
+        .collect();
```
Comment (zk-steve) [inline src/gen_const.rs]:
```suggestion
    let consts_unused: Vec<_> = table.keys()
        .into_iter()
        .filter(|item| !consts.contains(item))
        .collect();
```

Comment (zk-steve) [reply src/gen_const.rs]: separate function

File: src/gen_const.rs [hunk 4]
```diff
@@ -173,17 +189,22 @@ pub fn gen_consts(file_content: &str, table: &HashMap<String, ConstantValue>) ->
     let const_block = create_const_block(&consts, table);
```
Comment (zk-steve) [inline src/gen_const.rs:188]: call the new function

File: src/gen_const.rs (hunk 3 above)
Comment (andrew2003) [reply src/gen_const.rs]: done

File: src/gen_const.rs (hunk 4 above)
Comment (andrew2003) [reply src/gen_const.rs:188]: done

File: src/gen_const.rs (hunk 2 above)
Comment (andrew2003) [reply src/gen_const.rs]: done

File: Cargo.toml [hunk 5]
```diff
@@ -1,15 +1,24 @@
 [package]
+description = "Gen const for smart contract"
 edition = "2021"
-name = "orn"
+license = "MIT"
+name = "orn_cli"
```
Comment (zk-steve) [inline Cargo.toml]:
```suggestion
name = "orn-cli"
```

File: src/bin/orn.rs [hunk 6]
```diff
@@ -29,6 +31,15 @@ enum Commands {
 
 #[tokio::main]
 async fn main() {
+    // Will notify users in one day intervals if an update is available
```
Comment (zk-steve) [inline src/bin/orn.rs:34]: fix comment

**Review by zk-steve — APPROVED** [review]

**Pull request merged**